### ISSUE TITLE
Oculus touch configuration

### ISSUE THREAD

**Issue by Gacu666** [SuperV1234/quakevr#22]
Hi everyone :)

I really love the idea of Quake in VR, but alas I'm not as fluent in the arts of config fu and I'm struggling with my Oculus Rift CV1 and touch controllers. As of v.0.0.3 I can shoot, change weapons and jump (via button), the head tracking works great but all I'm missing is the movement and strafing.
Can anyone help a total n00b with proper configuration? A straight ass, idiot proof tutorial would be much appreciated :D

Cheers

**Comment by HunterMac** [comment]
Same issue here!

**Comment by Gacu666** [comment]
Ok I managed to figure it out. 
1) Start QuakeVR
2) Put on headset
3) Press home button on right touch controller
4) Navigate to SteamVR (it should be on the taskbar already)
5) Find settings -> Controller -> Manage Controller Bindings
6) The game should be highlighted and option "Edit this binding" or "Choose binding" should be available
7) Edit them yourself or choose one from already shared, I used "Stoni".
8) Play the game

They're not perfect, but I can move around, strafe and jump. Weapon switching is poor, but I'll try to change them to something more suitable.
